pkgs/development/tools/time-ghc-modules/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, sqlite
+, python3
+, coreutils
+, findutils
+, gnused
+}:
+
+stdenv.mkDerivation rec {
+  pname = "time-ghc-modules";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "codedownio";
+    repo = "time-ghc-modules";
+    rev = version;
+    sha256 = "0s6540gllhjn7366inhwa70rdnngnhbi07jn1h6x8a0pi71wdfm9";
+  };
+
+  nativeBuildInputs = [makeWrapper];
```

Comment:
Ater [ and before ] should be a space.

Reply:
if we were to enforce formatting, we would have a check as part of GHA. Otherwise I think we can ignore this kind of detail.